app/src/pages/settings/GlobalRetentionPolicyCard.tsx

```diff
@@ -0,0 +1,130 @@
+
+import { Card } from "@arizeai/components";
+
+import { Form, Input, Label, NumberField, Text } from "@phoenix/components";
+import { useNotifyError, useNotifySuccess } from "@phoenix/contexts";
+import { GlobalRetentionPolicyCardMutation } from "@phoenix/pages/settings/__generated__/GlobalRetentionPolicyCardMutation.graphql";
+import { GlobalRetentionPolicyCardQuery } from "@phoenix/pages/settings/__generated__/GlobalRetentionPolicyCardQuery.graphql";
+
+export const GlobalRetentionPolicyCard = () => {
+  const notifySuccess = useNotifySuccess();
+  const notifyError = useNotifyError();
+  const data = useLazyLoadQuery<GlobalRetentionPolicyCardQuery>(
+    graphql`
+      query GlobalRetentionPolicyCardQuery {
+        defaultProjectTraceRetentionPolicy {
+          cronExpression
+          id
+          name
+          rule {
+            __typename
+            ... on TraceRetentionRuleMaxDays {
+              maxDays
+            }
+          }
+        }
+      }
+    `,
+    {}
+  );
+  const [updateGlobalRetentionPolicy] =
+    useMutation<GlobalRetentionPolicyCardMutation>(graphql`
+      mutation GlobalRetentionPolicyCardMutation(
+        $input: PatchProjectTraceRetentionPolicyInput!
+      ) {
+        patchProjectTraceRetentionPolicy(input: $input) {
+          node {
+            id
+          }
+        }
+      }
+    `);
+
+  const { control, handleSubmit, watch } = useForm({
+    defaultValues: {
+      maxDays:
+        data.defaultProjectTraceRetentionPolicy?.rule?.__typename ===
+        "TraceRetentionRuleMaxDays"
+          ? data.defaultProjectTraceRetentionPolicy.rule.maxDays
+          : 0,
+    },
+  });
+
+  const id = data.defaultProjectTraceRetentionPolicy?.id;
+  const onSubmit = useCallback(
+    (form: { maxDays: number }) => {
+      updateGlobalRetentionPolicy({
+        variables: {
+          input: {
+            id,
+            rule: {
+              maxDays: {
+                maxDays: form.maxDays,
+              },
+            },
+          },
+        },
+        onCompleted: () => {
+          notifySuccess({
+            title: "Global retention policy updated",
+            expireMs: 5000,
+          });
+        },
+        onError: () => {
+          notifyError({
+            title: "Failed to update global retention policy",
+            expireMs: 5000,
+          });
+        },
+      });
+    },
+    [id, notifyError, notifySuccess, updateGlobalRetentionPolicy]
+  );
+
+  const debouncedSubmit = useMemo(() => {
+    return debounce(onSubmit, 800);
+  }, [onSubmit]);
+
+  const maxDays = watch("maxDays");
+
+  useEffect(() => {
+    debouncedSubmit({ maxDays });
+  }, [maxDays, debouncedSubmit]);
+
+  return (
+    <Card title="Global retention policy" variant="compact">
+      <Form onSubmit={handleSubmit(onSubmit)}>
+        <Controller
+          control={control}
+          name="maxDays"
+          render={({ field }) => (
+            <NumberField
```

Comment:
The number field feels a bit jammed in on the left. I wonder if you could make the layout a bit more "informative"